Use document mGet route in GET (#1519)
kuzzleio/kuzzle: doc/2/api/controllers/document/m-get/index.md

}
```

> You can either hit the route with `GET` verb

Reviewer: ```suggestion
You can also access this route with the `GET` verb:
```
